sdgx/data_models/inspectors/fixed_combination.py

```diff
@@ -0,0 +1,87 @@
+# FixedCombinationInspector
+from __future__ import annotations
+
+from typing import Any
+
+import pandas as pd
+
+from sdgx.data_models.inspectors.base import Inspector
+from sdgx.data_models.inspectors.extension import hookimpl
+
+
+class FixedCombinationInspector(Inspector):
+    """
+    FixedCombinationInspector is designed to identify columns in a DataFrame that have fixed relationships based on covariance.
+
+    Attributes:
+        fixed_combinations (dict[str, set[str]]): A dictionary mapping column names to sets of column names that have fixed relationships with them.
+        _inspect_level (int): The inspection level for this inspector, set to 70.
+    """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.fixed_combinations: dict[str, set[str]] = {}
+        """
+        A dictionary mapping column names to sets of column names that have fixed relationships with them.
+        """
+
+        self._inspect_level: int = 70
+        """
+        The inspection level for this inspector, set to 70. This attribute indicates the priority or depth of inspection that this inspector performs relative to other inspectors.
+        """
+
+    def fit(self, raw_data: pd.DataFrame, *args, **kwargs):
+        """
+        Fit the inspector to the raw data.
+
+        处理数值列和字符串列的固定组合关系:
+        - 数值列: 通过协方差矩阵计算相关性
+        - 字符串列: 通过值的一一对应关系判断
+        """
+        self.fixed_combinations = {}
+
+        # 1. 处理数值型列
+        numeric_columns = raw_data.select_dtypes(include=["int64", "float64"]).columns
+        if len(numeric_columns) > 0:
+            covariance_matrix = raw_data[numeric_columns].dropna().cov()
+            for column in covariance_matrix.columns:
+                related_columns = set(
+                    covariance_matrix.index[covariance_matrix[column].abs() > 0.9]
+                )
+                related_columns.discard(column)
+                if related_columns:
+                    self.fixed_combinations[column] = related_columns
+
+        # 2. 处理字符串列
+        string_columns = raw_data.select_dtypes(include=["object", "string"]).columns
```

Comment:
Considering that the enum values might be strings, floats, or integers, should the one-to-one correspondence between columns apply to all columns?
```suggestion
  # 2. 处理字符串列
        string_columns = raw_data.columns
```